docs/conf.py

```diff
@@ -40,14 +42,16 @@
 # ones.
 extensions = [
     'sphinx.ext.autodoc',
-    'sphinx.ext.imgconverter',
     'sphinx.ext.intersphinx',
     'sphinx.ext.todo',
     'sphinx.ext.coverage',
     'sphinx.ext.viewcode',
     'repoze.sphinx.autointerface',
 ]
 
+if pkg_resources.get_distribution('sphinx').version >= '1.6':
```

Comment:
I wasn't aware of this before, but I think we may be able to use [sphinx.version_info](https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.version_info) here.